libs/content/website/src/lib/content-wrapper/content-wrapper.tsx

```diff
@@ -57,7 +57,7 @@ export const ContentWrapperStyled = styled('article')<{fullWidth?: boolean}>`
         }
 
         & > ${QuoteBlockWrapper} {
-          grid-column-start: 5;
+          /* grid-column-start: 5; */
```

Comment:
should this be left like this?

Reply:
Oh good catch thanks! Was toggling it on and off to check if the margin was correct.